From 8b605dcea9888e0253f2c69cc53c4a2b8ad116f1 Mon Sep 17 00:00:00 2001
From: Rui Loura <rui@joyent.com>
Date: Thu, 12 Jul 2018 15:25:25 +0000
Subject: [PATCH] TRITON-587 net-agent needs updated regex for rack tagged
 admin nics Reviewed by: Cody Peter Mello <cody.mello@joyent.com> Approved by:
 Cody Peter Mello <cody.mello@joyent.com>

---
 lib/server-fsm.js | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/server-fsm.js b/lib/server-fsm.js
index 7019b98..92793cb 100644
--- a/lib/server-fsm.js
+++ b/lib/server-fsm.js
@@ -20,12 +20,12 @@ var mod_util = require('util');
 // --- Globals
 
 var VNIC_NAME_RE = /^([a-zA-Z0-9_]{0,31})[0-9]+$/;
-var ADMIN_TAG_RE = /^admin(_[a-zA-Z0-9_]+)?$/;
+var ADMIN_TAG_RE = /^ADMIN(_RACK_[A-Z0-9_-]+)?$/;
 
 // --- Internal helpers
 
 function isAdminLike(tag) {
-    return ADMIN_TAG_RE.test(tag);
+    return ADMIN_TAG_RE.test(tag.toUpperCase());
 }
 
 // --- Exports
-- 
2.21.0

